wcivf/apps/ppc_2024/management/commands/import_2024_ppcs.py

```diff
@@ -48,7 +48,9 @@ class CSVRow:
     def from_csv_row(cls, row: dict):
         party_id = clean_party_id(row.pop("Party ID", None))
         if not party_id:
-            raise BlankRowException()
+            with contextlib.suppress(BlankRowException):
```

Comment:
I think we still want to raise the exception here, but catch it where we call `from_csv_row` in the loop below. At that point we can use `suppress`